### ISSUE TITLE
Visualization

### ISSUE THREAD

**Issue by AlABarazi** [projectmesa/mesa#2193]
I have this code for real time visualization  : 
```
chart_element = ChartModule([
{"Label": "SystemLatency", "Color": "black"},
{"Label": "utilization", "Color": "brown"},
{"Label": "SystemLatencyUp", "Color": "grey"},
{"Label": "SystemLatencyDown", "Color": "pink"},
{"Label": "SystemLatencyMix", "Color": "orange"},
{"Label": "SystemLatencyH", "Color": "red"},
{"Label": "SystemLatencyM", "Color": "blue"},
{"Label": "SystemLatencyL", "Color": "green"},
{"Label": "utilizationH", "Color": "red"},
{"Label": "utilizationL", "Color": "green"},
{"Label": "utilizationM", "Color": "blue"},
{"Label": "computeSystemLatencyDown", "Color": "red"},
{"Label": "computeSystemLatencyUp", "Color": "blue"},
{"Label": "computeSystemLatencyMix", "Color": "green"},
], data_collector_name = "datacollector")
server = ModularServer(HRB, [text_element, canvas_element, chart_element],
                       "Vertical Logistics Simulation",model_params)


server.verbose = False
```

![Screenshot 2024-08-02 at 6 48 06 PM](https://github.com/user-attachments/assets/b84cd9a7-e77d-4a3b-9256-ee08a037b195)

 the correct label is not showing. All plot are labeled as "undefined" also the colors are not coorect. All are grey.

**Comment by rht** [comment]
Same reasoning as https://github.com/projectmesa/mesa/issues/2192#issuecomment-2266134029. Also, the visualization here is different from the newer Solara-based visualization tutorial. You are still using the old one which is being deprecated. If you want to keep using the old one, you have to use `mesa~=2.3.0`

**Comment by AlABarazi** [comment]
Thanks! 😎 Nice to add new visualization layer. The visualization in Mesa was really limited.

**Comment by AlABarazi** [comment]
problem solved using mesa=2.3.0. Thanks!
